config/karma.conf.js

```diff
@@ -163,14 +163,14 @@ export function getKarmaConfig(karma : Karma, cfg : Config) : KarmaConfig {
                     type: 'text'
                 },
                 {
-                    type:   'html',
+                    type:   'lcov',
```

Comment:
TIL that the `lcov` option includes both html and lcov reporting. https://karma-runner.github.io/0.8/config/coverage.html

Reply:
Sweet!